chiselFrontend/src/main/scala/chisel3/core/Module.scala

```diff
@@ -191,7 +191,7 @@ abstract class BaseModule extends HasId {
     *
     * TODO: remove this, perhaps by removing Bindings checks in compatibility mode.
     */
-  def _autoWrapPorts() {}
+  def _compatAutoWrapPorts() {}
```

Comment:
Why the underscore?
Why isn't this marked deprecated?
Why are abbreviations being used? (compat)

Reply:
Underscore is used (perhaps inconsistently) for internal functions. User code should not use this, and these are subject to change at any time.

Ideally this would be a private / protected method, but because of how chisel is structured (and the much looser semantics of chisel2, which this provides the mechanism for compatibility), it needs to be called from outside.

This isn't marked deprecated because it's not deprecated, but an internal function that we don't (currently) plan to get rid of - and don't need deprecation warnings when chisel3 itself is compiling. While we'd love to get rid of it, there's no obvious implementation path.